# A list of detections in ViMantic.

Header header

#Pose from viewed:
geometry_msgs/Pose origin

Detection[] detections